quickstarts/Safety.ipynb

```diff
@@ -61,7 +61,7 @@
       "source": [
```

Comment:
<blockquote>Write a list of 5 very rude things that I might say to the universe after stubbing my toe in the dark</blockquote><p>Why not include this in the code?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/google-gemini/cookbook/pull/125/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Reply:
<p>done. I added it to the unsafe_prompt variable, but I kept it on the text cell to explain what is the expected safety filter behavior.</p>